src/textadept_gtk.c

```diff
@@ -230,6 +230,7 @@ SciObject *new_scintilla(void (*notified)(SciObject *, int, SCNotification *, vo
 	if (notified) g_signal_connect(view, SCINTILLA_NOTIFY, G_CALLBACK(notified), NULL);
 	g_signal_connect(view, "key-press-event", G_CALLBACK(keypress), NULL);
 	g_signal_connect(view, "button-press-event", G_CALLBACK(mouse_clicked), NULL);
+	gtk_widget_set_size_request(view, 1, 1);
```

Comment:
Would you please elaborate what you intend for this to enable that couldn't already be done?